dataland-frontend/src/components/pages/ApiKeysPage.vue

```diff
@@ -254,7 +254,7 @@ export default defineComponent({
      *
      * @param daysValid the number of days the api key is valid for
      */
-    async generateApiKey(daysValid: number) {
+    async generateApiKey(daysValid?: number) {
```

Comment:
Why is this necessary?

Reply:
It says that "daysValid" can also have the value "undefined".   This is ok, because in that case the generated Api key should never expire.

Reply:
Ok, i was just wondering if there is a way to resolve the error message in the ide, as not to accumulate error messages.

Reply:
Oh I didn't know that there was an IDE error here. Let me investigate that.

Reply:
I'll check it

Reply:
Oh sorry, i confused this conversion with CreateLksgDataset.
This is ok then. Thank you and sorry for the confusion